CMakeLists.txt

```diff
@@ -167,39 +165,65 @@ add_executable(${TEST_NAME} ${Src})
 if(${hsa-runtime64_FOUND})
    target_link_libraries(${TEST_NAME} PRIVATE hsa-runtime64::hsa-runtime64)
 else()
-   target_link_libraries(${TEST_NAME} PRIVATE ${ROCR_LIB} ${ROCT_LIB} )
+   target_link_libraries(${TEST_NAME} PRIVATE ${ROCR_LIB} ${ROCT_LIB})
 endif()
 target_link_libraries(${TEST_NAME} PRIVATE c stdc++ dl pthread rt)
 
 # Update linker flags to include RPATH
 # Add --enable-new-dtags to generate DT_RUNPATH
 if( DEFINED ENV{ROCM_RPATH})
-   set ( CMAKE_EXE_LINKER_FLAGS "-Wl,--enable-new-dtags -Wl,--rpath,$ENV{ROCM_RPATH}" )
+   set ( CMAKE_EXE_LINKER_FLAGS "-Wl,--enable-new-dtags -Wl,--rpath,$ENV{ROCM_RPATH}")
```

Comment:
Do you mind changing the calls to cmake funcs consistent i.e. either have space or not have it